### PR TITLE
DDCNOS-4271 | Showing ServiceUnavailable page when RTI is down

### PR DESCRIPTION
**Pull request by shikkhasingh**
- [x] Unit tests passing
- [x] Coverage reached
- [x] Acceptance tests passing
- [x] Reviewed

### CODE REVIEW

File: app/controllers/PayeControllerHistoric.scala [hunk 1]
```diff
@@ -69,14 +69,24 @@ class PayeControllerHistoric @Inject()(
         hasTaxCodeRecordsInYearPerEmployment <- hasTaxCodeRecordsFuture
       } yield {
         implicit val user = request.taiUser
-        Ok(
-          views.html.paye.historicPayAsYouEarn(
-            HistoricPayAsYouEarnViewModel(taxYear, employments, hasTaxCodeRecordsInYearPerEmployment),
-            numberOfPreviousYearsToShow))
+        isRtiUnavailable(employments) match {
```
Comment (danewatts) [inline app/controllers/PayeControllerHistoric.scala]: Replace match for boolean with if else

File: app/controllers/PayeControllerHistoric.scala [hunk 2]
```diff
@@ -69,14 +69,24 @@ class PayeControllerHistoric @Inject()(
...
+              views.html.paye.historicPayAsYouEarn(
+                HistoricPayAsYouEarnViewModel(taxYear, employments, hasTaxCodeRecordsInYearPerEmployment),
+                numberOfPreviousYearsToShow))
+        }
       }
     }
   } recoverWith hodStatusRedirect
 
+  private def isRtiUnavailable(employments: Seq[Employment]): Boolean =
+    if (employments.head.annualAccounts.head.realTimeStatus.toString == "TemporarilyUnavailable")
```
Comment (danewatts) [inline app/controllers/PayeControllerHistoric.scala]: You dont need an if else to return a boolean. The statement inside the if returns a boolean. This is essentially writing ```if(true) true ese false```

File: app/controllers/PayeControllerHistoric.scala (hunk 1 above)
Comment (shikkhasingh) [reply app/controllers/PayeControllerHistoric.scala]: done

**Comment by danewatts** [comment]
When merging this PR, please Squash and merge

**Pull request merged**